packages/hint-compat-api/docs/architecture.md

```diff
@@ -0,0 +1,24 @@
+# Architecture
+
+This hint groups 4 different hints:
+
+* Hint for deprecated CSS features.
+* Hint for not broadly supported CSS features.
+* Hint for deprecated HTML features.
+* Hint for not broadly supported HTML features.
+
+It relies on the data from https://github.com/mdn/browser-compat-data to
+know the status of CSS and HTML features:
+
+* CSS: https://github.com/mdn/browser-compat-data/tree/master/css
+* HTML: https://github.com/mdn/browser-compat-data/tree/master/html
+
+## CLI Flow
+
+Every single hint takes the provided browserlist by the user (or the default
```

Comment:
We should link to https://webhint.io/docs/user-guide/configuring-webhint/browser-context/